releasenotes/notes/deprecate-span-pprint-master-101afa039a6bbaee.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    ``Span.pprint()`` is deprecated and will be removed in ``ddtrace==1.0``. Use ``Span.to_dict()`` to retrieve a map of Span attributes.
```

Comment:
```suggestion
    ``Span.pprint()`` is deprecated and will be removed in v1.0. Use ``Span.to_dict()`` to retrieve a map of Span attributes.
```

Also I think we can/should internalize `to_dict()` too so I'd hold off on recommending that.